config/initializers/sentry.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+Sentry.init do |config|
+  config.dsn = 'https://def412573e2b44d39e73a130b09bb25a@o1069899.ingest.sentry.io/4506591637471232'
```

Comment:
This looks right. If I remember correctly sentry support environment. In development is this logging to sentry development enviornment for testing?